src/renderer/components/EnhancedTable/EnhancedTableRow.js

```diff
@@ -144,18 +155,24 @@ const namespaceMenuItems = [
   },
 ];
 
-export const EnhancedTableRow = ({ extendedCloud }) => {
+export const EnhancedTableRow = ({ extendedCloud, withCheckboxes }) => {
+  const { getCheckboxValue, setCheckboxValue } = useCheckboxes(
+    makeCheckboxesInitialState(extendedCloud, extendedCloud.syncedNamespaces)
+  );
+  // show all namespaces if selectiveSync table or only syncedNamespaces in this main SyncView table
+  const usedNamespaces = withCheckboxes ? 'namespaces' : 'syncedNamespaces';
+
   const [onOpen, toggleMenu] = useState(false);
   const [isOpenFirstLevel, setIsOpenFirstLevel] = useState(false);
-  const [actualNamespaces, setActualNamespaces] = useState(
-    extendedCloud.syncedNamespaces
-  );
+  const [actualNamespaces, setActualNamespaces] = useState([
+    ...(extendedCloud[usedNamespaces] || []),
+  ]);
```

Comment:
```suggestion
  const [actualNamespaces, setActualNamespaces] = useState(extendedCloud[usedNamespaces]);
```

That's possible with #722...

Reply:
I'll update after you branch be merged

Reply:
It's merged!